cmd/artifact/command/add.go

```diff
@@ -196,7 +196,7 @@ func appendPushSubCommand(options *Options) *cobra.Command {
 			if err != nil {
 				return err
 			}
-			err = notifySlack(options, fmt.Sprintf(":white_check_mark: *Push* (%s:%s)", pushData.Image, pushData.Tag), slack.MsgColorYellow)
+			err = notifySlack(options, fmt.Sprintf(":white_check_mark: *Push* (<%s:image>)", fmt.Sprintf("%s:%s", pushData.Image, pushData.Tag)), slack.MsgColorYellow)
```

Comment:
Same here regarding the nested `Sprintf`s.

```suggestion
			err = notifySlack(options, fmt.Sprintf(":white_check_mark: *Push* (<%s:%s:image>)", pushData.Image, pushData.Tag), slack.MsgColorYellow)
```